configure

```diff
@@ -1197,8 +1197,10 @@ EOT
         open(my $fh, '<', $file) or die "ERROR: Could not open $file: $!";
         while (my $row = <$fh>) {
           chomp $row;
-          unless ($row =~ /^include/) {
-            my ($key, $value) = split(/=/, $row);
+          next if ($row =~ /^\s*#/);
+          $row =~ s/\s*#.*$//;
+          if ($row =~ /^\s*(\w+)\s*[:?+]=\s*(.*)$/) {
```

Comment:
Oops, my bad: The characters [:?+] shall be optional, therefore the line should be
```
if ($row =~ /^\s*(\w+)\s*[:?+]?=\s*(.*)$/) {
```
Notice the added "?" after the character group.